src/Sylius/Bundle/ApiBundle/Validator/Constraints/ChangingItemQuantityInCart.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class ChangingItemQuantityInCart extends Constraint
```

Comment:
```suggestion
final class ChangedItemQuantityInCart extends Constraint
```
would be more consistent with other constraints